frontend/packages/console-shared/src/components/modals/ConsolePluginModal.tsx

```diff
@@ -41,7 +41,7 @@ export const ConsolePluginModal = withHandlePromise((props: ConsolePluginModalPr
     <form onSubmit={submit} name="form" className="modal-content">
       <ModalTitle>
         {csvPluginsCount > 1
-          ? t('console-shared~Console plugin enablement - {{plugin}}', { pluginName })
+          ? t('console-shared~Console plugin enablement - {{pluginName}}', { pluginName })
```

Comment:
I'd suggest this instead so we don't have to re-translate the messages:

```suggestion
          ? t('console-shared~Console plugin enablement - {{plugin}}', { plugin: pluginName })
```

Reply:
updated